### PR TITLE
Add WasmGC spec

### PR DESCRIPTION
**Pull request by tomayac**
https://webassembly.github.io/spec/core/

### CODE REVIEW

File: specs.json [hunk 1]
```diff
@@ -452,6 +452,10 @@
     "url": "https://webassembly.github.io/exception-handling/js-api/",
     "forkOf": "wasm-js-api-2"
   },
+  {
+    "url": "https://webassembly.github.io/gc/core/",
+    "forkOf": "wasm-core"
```
Comment (tomayac) [inline specs.json:457]:
```suggestion
    "forkOf": "wasm-core-1" 
```

File: specs.json [hunk 2]
```diff
@@ -452,6 +452,10 @@
     "url": "https://webassembly.github.io/exception-handling/js-api/",
     "forkOf": "wasm-js-api-2"
   },
+  {
+    "url": "https://webassembly.github.io/gc/core/",
+    "forkOf": "wasm-core-1"
```
Comment (dontcallmedom) [inline specs.json:457]:
```suggestion
    "forkOf": "wasm-core-1"
```
trailing white space triggers lint error

File: specs.json [hunk 3]
```diff
@@ -452,6 +452,10 @@
...
```
Comment (dontcallmedom) [inline specs.json:457]:
```suggestion
    "forkOf": "wasm-core-2"
```
it seems to be a fork of level 2 - but then for some reason, it looks like we don't have level 2 in specs.json - @tidoust, does that ring a bell?

Comment (tidoust) [reply specs.json:457]: See https://github.com/w3c/browser-specs/pull/593 but we should revisit that. I created #1059 accordingly.

File: specs.json [hunk 4]
```diff
@@ -452,6 +452,10 @@
     "url": "https://webassembly.github.io/exception-handling/js-api/",
     "forkOf": "wasm-js-api-2"
   },
+  {
+    "url": "https://webassembly.github.io/gc/core/",
```
Comment (tidoust) [inline specs.json:456]: We're going to need that to point to the Bikeshed version, as our parsing tools cannot parse this format.

```suggestion
    "url": "https://webassembly.github.io/gc/core/bikeshed/",
```

**Review by tidoust — APPROVED** [review]

**Pull request merged**